pages/product-selection-opensuse-page.ts

```diff
@@ -0,0 +1,33 @@
+import { test, expect } from '@playwright/test';
+
+export class ProductSelectionPage {
+    readonly page: Page;
+    readonly productSelectionText: Locator;
+    readonly opensuseTumbleweedCheck: Locator;
+    readonly opensuseLeapCheck: Locator;
+    readonly selectButton: Locator;
+
+    constructor(page: Page) {
+        this.page = page;
+        this.productSelectionText = page.getByText('Product selection');
+        this.opensuseTumbleweedCheck = page.getByLabel('openSUSE Tumbleweed');
+        this.opensuseLeapCheck = page.getByLabel('openSUSE Leap 16.0');
+        this.selectButton = page.getByRole('button', { name: 'Select' });
+    }
+
+    async expectProductSelection() {
+        await expect(this.productSelectionText).toBeVisible(true);
+    }
+
+    async choose_opensuse_tumbleweed() {
```

Comment:
sorry, I fell in the trap myself recommending you names, please camel-based names, chooseOpensuseTumbleweed()

Reply:
Updated